packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -682,12 +684,42 @@ export function mixinSynchronization<
 				);
 			};
 			break;
+
+		case "partialSynchronization":
+			// passing 1 here causes infinite loops. passing close to 1 is wasteful
+			// as synchronization + eventual consistency validation should be idempotent.
+			// 0.5 is arbitrary but there's no reason anyone should want a probability near this.
+			assert(
+				validationStrategy.probability < 0.5,
+				"Use a lower synchronization probability.",
+			);
+			generatorFactory = (): Generator<TOperation | Synchronize, TState> => {
+				const baseGenerator = model.generatorFactory();
+				return async (state: TState): Promise<TOperation | Synchronize | typeof done> => {
+					if (!state.isDetached && state.random.bool(validationStrategy.probability)) {
+						const connectedClients = new Set(
+							state.clients
+								.filter((client) => client.containerRuntime.connected)
+								.filter(() =>
+									state.random.bool(validationStrategy.clientProbability),
+								)
+								.map((client) => client.containerRuntime.clientId),
```

Comment:
shouldn't this pick a subset of connected client, and not all of them? otherwise its not different than full synchronize, as offline clients won't sync anyway.

Reply:
The filter statements do exactly that, first filters to connected clients and then uses the clientProbability to decide whether or not a client should be selected.